notebooks/234-blip/Untitled.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Why two decoders are needed? I'm not sure I fully understand this, maybe need some explanation?</p><p>Is it to handle previous input as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>this also will be covered in next commit</p><p> </p>

Reply:
added